### PR TITLE
Room assignment task

### PR DESCRIPTION
**Pull request by katelynbottcher**
https://github.com/Watts-Lab/task-mapping/issues/75

### CODE REVIEW

File: Room assignment task.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+# Summary
+Participants had to assign students to rooms to maximize student utility.
```
Comment (sarikasu) [inline Room assignment task.md:2]: should this include that they needed to respect constraints?

File: Room assignment task.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+> > > 
+> > > In each room assignment problem, a “utility table” was presented, providing participant(s) with the information on students’ ratings/preferences (between 0 and 100) to each of the M rooms indicating how satisfied they would be if being assigned to the room.
+> > > 
+> > >  At any time during the allotted 5-minute period for a task, the participant could push a button to submit her solution and move on to the next task, or the participant would be automatically redirected to the next task when the 5-minute timer was up
+> > > 
+> Phase Two: 828 of the original 1200 participants completed 5 more room tasks varying in very low, low, moderate, high, and very high complexity, with all tasks timing out at 10 mins. 
+> > > Based on each individual's score in task one, they were categorized into blocks by skill level and social perceptiveness (HH, MH, LH, HL, ML, LL) 
+> > > 
+> > > Within these categories, the participants were then separated either into groups, or remained individual.
+> > >
```
Comment (sarikasu) [inline Room assignment task.md:47]: individuals were randomized into groups

File: Room assignment task.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+> > > > Speed (“tortoises” vs. “hares”)
+> > > > 
+> > > > Preference for efficiency vs. perfection (“pragmatist” vs. “perfectionist”)
+> > > > 
+> > > > Tolerance for constraint violation (low vs. high)
+> > > > 
+> > > > Preference for optimization vs. constraint resolution (“optimizer” vs. “resolver”): if the participant assigned students then worked out conflicts they were in the "optimize first" category, and if they worked out conflicts before moving on to assigning more students then they were in the "satisfy first" category.
+>
+> Phase Two:
+> > Performance was calculated by considering the normalized score the individual or group obtained on this task, defined as score on task T/ max score for task T.
```
Comment (sarikasu) [inline Room assignment task.md:93]: Should we add the task sequence was randomized and varied in complexity?

File: Room assignment task.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
+# Summary
+Participants had to assign students to rooms to maximize student utility.
+
+# References
+> Main paper: https://psyarxiv.com/ra9qy/
+
```
Comment (sarikasu) [inline Room assignment task.md:6]: The researchers have a really good overview of the experiment here: https://github.com/amaatouq/room-assignment/blob/master/Readme.md

Comment (sarikasu) [reply Room assignment task.md:6]: they also include all of their materials on this page: https://github.com/amaatouq/room-assignment

Comment (katelynbottcher) [reply Room assignment task.md:7]: Great find! What materials are you referring to?

**Review by sarikasu — APPROVED** [review]

**Pull request merged**